api/room/schema.py

```diff
@@ -67,6 +68,7 @@ class Arguments:
         image_url = graphene.String()
         location_id = graphene.Int(required=True)
         calendar_id = graphene.String()
+        structure_id = graphene.Int()
```

Comment:
Don't you think is in necessary to make this field a required field?

Reply:
Thanks,  @Bonifase  for pointing this out.  I have changed it.